docs/source/samples/FeatureTracker/feature_tracker.rst

```diff
@@ -0,0 +1,42 @@
+Feature Tracker
+===============
+
+Example shows capabilities of :code:`:ref:`FeatureTracker``. It detects features and tracks them.
```

Comment:
and tracks them `between consecutive frames using optical flow by assigning unique ID to matching features`